src/Wallabag/CoreBundle/Resources/views/base.html.twig

```diff
@@ -113,5 +113,6 @@
                 <p>{% trans %}powered by{% endtrans %} <a href="http://wallabag.org">wallabag</a></p>
             </footer>
         {% endblock %}
+        <div id="alphabar">You're trying wallabag v2, which is in alpha version. If you find a bug, please have a look to <a href="https://github.com/wallabag/wallabag/issues">our issues list</a> and <a href="https://github.com/wallabag/wallabag/issues/new">open a new if necessary</a>.</div>
```

Comment:
Hum I don't really like this hard coded way, can we defined that in the parameters.yml file? So when there is something in to display, we just have to fill the parameter and the alpha message will be displayed